### ISSUE TITLE
Unexpected templating error for system probe

### ISSUE THREAD

**Issue by kevinsupendi** [DataDog/ansible-datadog#213]
Hi, had issues related to system probe templating, saying that unexpected argument 'first' for indent(width=2, first=True). I think the argument should be indentfirst instead of first.  @kevinconaway #210 [link](https://github.com/DataDog/ansible-datadog/blob/master/templates/system-probe.yaml.j2)

![image](https://user-images.githubusercontent.com/10279776/66998782-0a33bb80-f0ff-11e9-9a46-142e96d5b016.png)

**Comment by kevinconaway** [comment]
> I think the argument should be indentfirst instead of first.

What version of jinja are you using?

It looks like the argument has been renamed to `first` in [jinja 2.9](https://jinja.palletsprojects.com/en/2.9.x/templates/) onwards.  Let me see if there is a way to support both.

**Comment by hush-hush** [comment]
Hi @kevinsupendi,

Which version of ansible are you using ? From my understanding Ansible `2.6` use jinja2 `2.10` on a fresh install which should have the `first` feature in.

Could you also tell us how you installed ansible (pip ? os packge ?). Not sure ansible does pin jinja2 when installing with pip.

Thanks

**Comment by kevinsupendi** [comment]
hi, thank you for your responses, after I checked my Ansible AWX setup, I found that the worker VM has 2.10.1 installed, but the container that run the task has jinja 2.8 instead, which explains why the datadog playbook failed. Thanks @kevinconaway for the information! I'll make sure to update the jinja2 version in the container as well.

![image](https://user-images.githubusercontent.com/10279776/67067371-4ec36380-f1a0-11e9-8465-732f0e1bafbb.png)


> > I think the argument should be indentfirst instead of first.
> 
> What version of jinja are you using?
> 
> It looks like the argument has been renamed to `first` in [jinja 2.9](https://jinja.palletsprojects.com/en/2.9.x/templates/) onwards. Let me see if there is a way to support both.
